ansible/roles/ocp-workload-cnad-app-modernization/files/crw-operator.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: codeready-workspaces
+  namespace: codeready-workspaces
+spec:
+  channel: latest
+  installPlanApproval: Automatic
+  name: codeready-workspaces
+  source: redhat-operators
```

Comment:
You definitely want an Operator Snapshot here to prevent accidental breakage upon CRW releases.... the `ocp4_workload_codeready_workspaces` already has that enabled.